creator-node/src/services/sync/processSync.js

```diff
@@ -106,7 +106,7 @@ async function processSync (serviceRegistry, walletPublicKeys, creatorNodeEndpoi
 
     try {
       // Attempt to connect directly to target CNode's IPFS node.
-      await _initBootstrapAndRefreshPeers(serviceRegistry, logger, body.data.ipfsIDObj.addresses, redisKey)
+      _initBootstrapAndRefreshPeers(serviceRegistry, logger, body.data.ipfsIDObj.addresses, redisKey)
```

Comment:
this makes sense, @dmanjunath just confirming this is to minimize latency?

thinking more on this, we can prob reduce frequency of running this cmd as well, but fine for now. prob would be good to add duration to the log though

side note @isaacsolo - if we are going to not await an async function, we should add an explicit comment to that effect

Reply:
yeah to minimize latency @SidSethi, this was in the card you made

Reply:
hahaha lovely. ofc it was. cool 👍